src/services/backends/backend.service.ts

```diff
@@ -20,7 +24,12 @@ export const requestBackend = (ast: AstTopLayer, updateResponse: (response: Map<
 
     const library = buildLibrary(`${cql}`);
     const measure = buildMeasure(library.url, measures);
-    const query = { lang: "cql", lib: library, measure: measure };
+    query = { lang: "cql", lib: library, measure: measure };
+
+    // Fallback to AST
+    } else /*if (import.meta.env.VITE_BACKEND_FORMAT === "ast")*/ {
+        query = { lang: "ast", payload: btoa(decodeURI(JSON.stringify({ast: ast, id: queryId.concat("__search__").concat(queryId) }))) };
+    }
 
 
     let backendUrl: string = ""
```

Comment:
Semicolon missing?

Reply:
This semicolon is optional, however, I think it's always a good idea to add it.
For claritie's sake added in c7f69d49eb386f7981a9930956a3